### PR TITLE
GitHub Discussions for questions and translations

### LINKED ISSUE #1571: Adopt Category Forms for Community Questions
### Addressing a Problem?

`xclim` is becoming more and more popular (amazing! :100: ) and so we have been receiving more and more questions from users that fall more into the support category (for approaches and workflows). In some instances, these issues have revealed latent bugs in our code base and in others the answers have been helpful in clarifying a few "how-to"'s and "Best Practices".

It would be great if we could promote support questions to the issues (in the case of bugs) and be able to more easily workshop with users needing general support without it affecting our issue metrics. 

### Potential Solution

In our last internal meeting, we discussed the potential approach of rallying support questions into a more wiki-like environment. Discourse and GitHub Discussions were considered; Ultimately, for `xclim`-specific questions, we want to re-open Discussions with some templates for users with general questions.

As of 2023, this is now possible. This proposal is to move `support` questions into GitHub Discussions, and migrate/cross-post `Discussion` questions if they require code changes into `Issues`.

### Additional context

https://github.blog/2023-01-09-github-discussions-just-got-better-with-category-forms/

### Contribution

- [X] I would be willing/able to open a Pull Request to contribute this feature.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #1571 
- [ ] Tests for the changes have been added (for bug fixes / features)
  - [x] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [x] CHANGES.rst has been updated (with summary of main changes)
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added

### What kind of change does this PR introduce?

* Migrates the Question/Support Issue Template to a GitHub Discussion Template
* Adds a new page for coordinating translation efforts 
* Added a link to PAVICS for PAVICS-specific questions. Once we have a `Discourse` up and running, we'll be able to funnel all PAVICS-specific question there.

### Does this PR introduce a breaking change?

Not really. This makes it so that the following can happen more easily:
- Answered support requests will more easily be found by users with similar usage questions. This will make it simpler for users as `answering` a discussion post does not hide the post like `closing` an issue does.
- Translation efforts can now more easily be coordinated outside of Pull Requests for users wishing to contribute translations in languages other than English or French. In a previous attempt, the major translation effort was performed in a Pull Request with 100+ comments in it and was impossible to navigate near the end. I've noticed similar translation efforts (using GitHub Discussions) organized in other projects, such as `FastAPI`  (https://github.com/tiangolo/fastapi/discussions/categories/translations).

### Other information:

https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms

### CODE REVIEW

**Comment by bzah** [comment]
I would suggest to also update the Readme, in particular the [Contributing to xclim ](https://github.com/Ouranosinc/xclim#contributing-to-xclim) section in order to guide user to the github discussion forum instead of gitter.

**Comment by Zeitsperre** [comment]
> I would suggest to also update the Readme, in particular the [Contributing to xclim ](https://github.com/Ouranosinc/xclim#contributing-to-xclim) section in order to guide user to the github discussion forum instead of gitter.

That's a good idea. I don't seem to have admin access over the Gitter room, so I can't make changes. Does anyone know who created the room? @huard?

**Comment by Zeitsperre** [comment]
Update: so in order to retire the Gitter room, I need to manually kick everyone out of it (and ask any existing moderators to leave). It doesn't seem like there's an easy way to close down a room other than booting everyone and making it private.

I've saved the chat log (it wasn't very long, TBF), so once this Pull Request is in, I'll start the process of winding the Gitter down.

File: .github/DISCUSSION_TEMPLATE/questions.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      label: Setup Information
+      description: |
+        What software versions are you running? Example:
+        * Xclim version: 0.55.0-gamma
+        * Python version: 4.2
+        * Operating System: Nutmeg Linux 12.34 | macOS 11.0 "Redmond"
+      value: |
+        * Xclim version:
+        * Python version:
+        * Operating System:
```
Comment (huard) [inline .github/DISCUSSION_TEMPLATE/questions.yml:23]:
```suggestion
        What xclim version are you running? 
      value: |
        * Xclim version:
```

**Review by huard — APPROVED** [review]

File: .github/DISCUSSION_TEMPLATE/questions.yml (hunk 1 above)
Comment (Zeitsperre) [reply .github/DISCUSSION_TEMPLATE/questions.yml:23]: @huard would you be amenable to something more professional, but still asking for the same information? I find that knowing whether the user is running an older version of Python or on Windows is important to identifying edge cases.

```suggestion
        What is your system information?
      value: |
        * Xclim version:
        * Python version:
        * Operating System:
```

**Pull request merged**